typos.toml

```diff
@@ -0,0 +1,6 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+[default.extend-words]
+enew = "enew"
+fo = "fo"
```

Comment:
Could you explain these two word additions?

Reply:
As you can see above, they are in the section of `[default.extend-words]`, which means these words are supposed to be correct (they may be not valid English words, but they are valid keywords in a certain programming language, or a commit hash, etc). You can see https://github.com/crate-ci/typos/blob/master/docs/reference.md for more infomation.

Reply:
Yeah, so can you explain why these words shouldn't be considered wrong? Maybe my original question wasn't clear 😅

Reply:
Is this checking the words in the code as well or just the comments?

Reply:
Checking all

Reply:
Hm I'll have to think about this and get back to you 🤔 also what about other things we have that aren't English words? Like `ffi`? Why is that not triggering it

Reply:
> Yeah, so can you explain why these words shouldn't be considered wrong? Maybe my original question wasn't clear 😅

Oh, I see. `enew` is a vim command and `fo` is a keyboard shortcut, so I think they should be thought as correct.

Reply:
There are a lot more non English words in the code than just those two. I will review this sometime next week and get back to you. Thanks for having this idea :) ideally we would be able to just spell check the comments

Reply:
> > Yeah, so can you explain why these words shouldn't be considered wrong? Maybe my original question wasn't clear 😅
> 
> Oh, I see. `enew` is a vim command and `fo` is a keyboard shortcut, so I think they should be thought as correct.

See https://github.com/crate-ci/typos/#false-positives

Reply:
Maintaining these "corrections" will get unruly and unmanageable in the future.

Reply:
> Hm I'll have to think about this and get back to you 🤔 also what about other things we have that aren't English words? Like `ffi`? Why is that not triggering it

I haven't read its design docs and I just use it to find out potential misspelling. As far as I see, it's not something like dictionary and list all non-English-valid words.

Reply:
> Maintaining these "corrections" will get unruly and unmanageable in the future.

We can skip some certain files in `[files]` section like `mappings.lua` to solve it.

Reply:
There are many many non English words throughout all of the files. I'll get back to you in - a few days.

Reply:
> There are many many non English words throughout all of the files. I'll get back to you in - a few days.

And we can see how https://github.com/crate-ci/typos/ works. Maybe it will answer your questions.